synapseutils/migrate_functions.py

```diff
@@ -608,9 +633,18 @@ def migrate_indexed_files(
                     from migrations
                     where
                         status = ?
-                        and ((id > ? and type in (?, ?))
-                            or (id = ? and type = ? and version is not null and version > ?)
-                            or (id = ? and type = ? and (row_id > ? or (row_id = ? and col_id > ?))))
+                        and (
+                                (
+                                    ((id > ? and type in (?, ?))
+                                    or (id = ? and type = ? and version is not null and version > ?)
```

Comment:
Can use named or positional but not both, which is a bit inconvenient since there are a couple dynamically sized lists of values that isn't easily named. Using named literals with some inlined values, the inlined values are file handle ids so should have known formatting and not result in formatting issues due to unexpected characters etc.